src/views/choose-appeal-reason.njk

```diff
@@ -0,0 +1,59 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk'     import govukBackLink %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from 'govuk/components/radios/macro.njk' import govukRadios %}
+
+{% block pageTitle %}
+  You must tell us the reason for the appeal
+{% endblock %}
+
+{% block backLink %}
+  {{ 
+    govukBackLink({
+      text: 'Back',
+      href: navigation.back.href
```

Comment:
I do not think story defines that at this point and there are navigations stories in the backlog. I would not implement it in the similar stories.

Reply:
I can't figure out how to implement validation without extending the SafeNavigationBaseController, which in turn requires the navigation object. My thought process is that I'd rather do a small amount of future work if it means I can avoid a significantly greater amount of work of figuring out how to how to do something in a way that will be completely reworked anyway as soon as the navigation tickets come into play.

Reply:
Resolved: Controller now extends BaseAsyncController so navigation object is no longer required.